armi/reactor/reactors.py

```diff
@@ -435,12 +435,27 @@ def removeAssembly(self, a1, discharge=True):
         else:
             self._removeListFromAuxiliaries(a1)
 
-    def removeAssembliesInRing(self, ringNum):
+    def removeAssembliesInRing(self, ringNum, forceHexRing=False):
         """
         Removes all of the assemblies in a given ring
+
+        Parameters
+        ----------
+        ringNum : int
+            The ring to remove
+
+        forceHexRing : bool, optional
+            True ~ if you want to remove assemblies in square or hex ring from a squre or hex reactor.
+            False ~ if you want to remove assemblies in a circular ring.
```

Comment:
Actually on second thought, I think a better arg would be called `overrideCircularRingMode=None` which would just get passed directly down to `getAssembliesInRing`. Then you would also add that same arg to `getAssembliesInRing`, which would do a `is not None` check and if it's not `None` override the `_circularRingMode` param in that method. 

Otherwise we're mixing levels of abstraction in this method. Furthermore, this fix will enable this override to any other client of getAssembliesInring.

Reply:
Done